lib/code_corps/stripe_service/util.ex

```diff
@@ -10,6 +10,53 @@ defmodule CodeCorps.StripeService.Util do
   """
   def transform_map(api_map, mapping), do: mapping |> Enum.reduce(%{}, &map_field(&1, &2, api_map))
 
+  def transform_attributes(attributes, mapping) do
+    attributes_map = map_keys_to_atoms(attributes)
+    mapping |> Enum.reduce(%{}, &map_attribute(&1, &2, attributes_map))
+  end
```

Comment:
Might not be a bad Idea to have a unit test for this specifically. Separate issue might be good enough for now, though.